streamwebs_frontend/streamwebs/templates/streamwebs/create_site.html

```diff
@@ -3,22 +3,116 @@
 {% load staticfiles %}
 
 {% block scripts %}
-  <script src="//maps.google.com/maps/api/js?callback=initMap" async
-          defer></script>
-  <script type="application/javascript"
-          src="{% static 'streamwebs/js/create_site.js' %}"></script>
+<script type="text/javascript" src="http://maps.google.com/maps/api/js?key={{ maps_api }}"></script>
+<script type="application/javascript" src="{% static 'streamwebs/js/create_site
+.js' %}"></script>
+<script>
+var markers = [];
+var map;
+
+$(document).ready(function() {
+  document.getElementById("site_form").reset();
+  var myLatLng = new google.maps.LatLng(43.8041, -120.5542);
+  map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 7,
+    center: myLatLng,
+    streetViewControl: false
```

Comment:
please add the "mapTypeId: 'terrain'" option here, I think terrain should be the default map type, as it is in the old system